plugins/BEdita/Core/tests/TestCase/Model/Entity/FolderTest.php

```diff
@@ -169,4 +169,40 @@ public function testListAssociations()
         $folder = $folder->jsonApiSerialize();
         static::assertArrayHasKey('parent', $folder['relationships']);
     }
+
+    /**
+     * Data provider for `testGetPath()`
+     *
+     * @return array
+     */
+    public function getPathProvider()
+    {
+        return [
+            'root' => [
+                '/11',
+                11,
+            ],
+            'subfolder' => [
+                '/11/12',
+                12,
+            ]
+        ];
+    }
+
+    /**
+     * Test getter for `path`
+     *
+     * @return void
+     *
+     * @dataProvider getPathProvider
+     * @covers ::_getPath()
+     */
+    public function testGetPath($expected, $id)
+    {
+        $folder = new Folder();
+        static::assertNull($folder->path);
```

Comment:
[minor] I'd split this test into its own test case, and use `$this->Folders->newEntity()` instead.

Reply:
yeah, I think it's better :+1: